README.md

```diff
@@ -190,7 +190,11 @@ SD + APP      | Yes       | **See notes 1 and 2 below**
 was added in nrfutil 3.1.0 and is required since 3.2.0 in case the package should contain SD (+ BL) + APP. Also, since version 3.2.0 the new ID is copied to `--sd-req` list so that
 in case of a link loss during APP update the DFU process can be restarted. In that case the new SD would overwrite itself, so `--sd-req` must contain also the ID of the new SD.
 
-The boolean option '--zigbee' enables the generation of Zigbee update file in addition to the zip package. The following example demonstrates the generation of such update file:
+Boot validation for a SD or APP update can be activated by setting the `--sd-boot-validation` or '--app-boot-validation' to the preferred validation method. When boot validation is set, the bootloader will store the signature of the SD or APP in the bootloader setting page and verify the signature with the firmware in flash on every boot. This is only supported by bootloaders using bootloader settings version 2, see 'settings' for more information.
```

Comment:
Maybe mention that the boot-validation arguments can be used both with package generation and with settings generation

Reply:
Added